### ISSUE TITLE
Emails to help address are bouncing

### ISSUE THREAD

**Issue by HeatherBrooks** [ODM2/ODM2DataSharingPortal#309]
A user notified me that he received a bounce message when he emailed requesting a new sensor type. This is because the emails are being directed to ww-data-portal-help@googlegroups.com (see bottom of form in screen shot) but only group members can send directly to a Google Group. I can set up an address (e.g. help@monitormywatershed.org), have it forward to me, and then I'll forward it to the Google Group. (This two-step process is similar to how we handle contact form submissions at https://wikiwatershed.org/help/#contact.)

Please let me know what address you'd like me to set up.
![add-new-sensor-form](https://user-images.githubusercontent.com/9368629/46971445-f687ec00-d089-11e8-8ae0-d4cc51660c87.png)

**Comment by aufdenkampe** [comment]
@HeatherBrooks, Google groups used to be able to be set to receive emails from the public. I'm nearly certain that we just need to change a setting in the Google Group.

**Comment by aufdenkampe** [comment]
@HeatherBrooks, for this group at https://groups.google.com/d/forum/ww-data-portal-help, I just modified *Permissions>Basic Permissions* to allow "Anyone on the web" to Post questions and View topics.

Only members of the group can respond.

I also reset the "Topic types" to default to "Questions", rather than "Discussions", under *Information>General information*.

Have the user try again to email us.

**Comment by HeatherBrooks** [comment]
@aufdenkampe, thanks for updating the settings. You probably saw in your inbox, but Neil Hancock successfully emailed the Google Group this morning. He reports: "I can now see messages in https://groups.google.com/forum/#!forum/ww-data-portal-help." Is a public forum, separate from the forums on EnviroDIY.org, what you had in mind?

**Comment by SRGDamia1** [comment]
Please, not another forum!  Just one should be sufficient.

**Comment by aufdenkampe** [comment]
@HeatherBrooks, I don't want another general forum either. My intent was to set it up as a public help email system that very specifically focused on the Portal, not other EnviroDIY stuff. I thought that keeping old responses public could be good. I'll look at it again. I thin both you and @SRGDamia1 have access to change settings.

**Comment by horsburgh** [comment]
@HeatherBrooks and @aufdenkampe - I'm assigning this one to you. If this has already been resolved, you can just close this issue.

**Comment by aufdenkampe** [comment]
This has been resolved and is working.

**Comment by HeatherBrooks** [comment]
I don't think this is resolved ... no action has been taken since the Google Group was set to public and NeilH (and a spammer) began posting to the associated forum. We discussed this at the Stroud Center -- we really don't want another forum to have to respond to -- so we would prefer to make the Google Group private again and use the approach I outlined when I opened this issue.

**Comment by HeatherBrooks** [comment]
Please change the email address for add new sensor requests from ww-data-portal-help@googlegroups.com to help@monitormywatershed.org.

**Comment by aufdenkampe** [comment]
@HeatherBrooks, following your suggestion, I just edited things at https://groups.google.com/d/forum/ww-data-portal-help, where I modified Permissions>Basic Permissions to only allow "All members of the group" to view topics. I left "Anyone on the web" to Post questions.

This way it should not allow people to see it as a forum, but will still accept (and forward) any emails sent to that address (which could be helpful at least until the release).
